### PR TITLE
Lut 20544

### PR DESCRIPTION
**Pull request by Timothee-Hx**
When the user changes the order of a question to export with the select an http request is made thought a jsp

### CODE REVIEW

File: src/java/fr/paris/lutece/plugins/forms/business/IQuestionDAO.java [hunk 1]
```diff
@@ -198,4 +198,25 @@ public interface IQuestionDAO
...
+    /**
+     * Load the data of the question objects by form id between 2 export_display_order and returns them as a list
+     * 
+     * @param nIdForm
+     * @param nExportOrder1
+     * @param nExportOrder2
+     * @param plugin
+     * @return
+     */
+	List<Question> selectQuestionsListByFormIdBetweenExportOrders(int nIdForm, int nExportOrder1, int nExportOrder2,
```
Comment (seboo) [inline src/java/fr/paris/lutece/plugins/forms/business/IQuestionDAO.java:220]: What's the use of this method ?

File: src/test/java/fr/paris/lutece/plugins/forms/business/form/service/MultiviewFormServiceTest.java [hunk 2]
```diff
@@ -69,7 +69,7 @@ protected void tearDown( ) throws Exception
     }
 
     /**
-     * Test of the method {@link fr.paris.lutece.plugins.forms.service.MultiviewFormService#findActiveFormPanel(java.util.List)}
```
[the remark this answers is not in the document]
Comment (Timothee-Hx) [reply src/test/java/fr/paris/lutece/plugins/forms/business/form/service/MultiviewFormServiceTest.java]: Oups, this wasn't suppose to be part of my commit !

File: src/java/fr/paris/lutece/plugins/forms/business/IQuestionDAO.java (hunk 1 above)
Comment (Timothee-Hx) [reply src/java/fr/paris/lutece/plugins/forms/business/IQuestionDAO.java]: Oups, this wasn't suppose to be part of my commit

**Comment by Timothee-Hx** [comment]
Correction for 20544: from commit on the 12 of july. 
For this issue, just the files FormExportJspBean.java and manage_export.html need to be merged.

**Comment by seboo** [comment]
The sort algorythm should be more readable, i will create another issue about it.
Please prefix every commit by his Redmine reference, and isolate formatting in a separate commit.
I will sqash those ones to get a clean commit label .

**Pull request merged**